README.md

```diff
@@ -1,16 +1,37 @@
 # dcwrapper
 Dakota-Cyclus Wrapper 
-This repository contains examples of Cyclus sensitivity analysis studies conducted with a Dakota wrapper. Cyclus interfaces with Dakota using a Python interface. 
+This repository contains examples of Cyclus sensitivity analysis 
+studies conducted with a Dakota wrapper. 
+Cyclus interfaces with Dakota using a Python interface. 
 
-To run the simple test case, go into the dakota/test/ directory and run dakota -i dakota_test.in -o dakota_test.out
+To run the simple test case, run the following from this directory: 
+```
+bash
+cd dakota/test/ 
+dakota -i dakota_test.in -o dakota_test.out
+```
+
+The structure of each sensitivity analysis study in this repository 
+is as follows: 
+1. For each sensitivity analysis study, their Dakota input files goes 
```

Comment:
"For each sensitivity analysis ... " is repeated in lines 14, 16, and 20 - can probably remove 16 and 20.